### PR TITLE
Use appropriate labels and placeholders

### PR DESCRIPTION
**Pull request by silverbackdan**
When the form is loaded without ajax on the form post, the label should be correct immediately. See https://github.com/Sylius/Sylius/issues/8152

| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | fixes https://github.com/Sylius/Sylius/issues/8152, partially #Y, mentioned in #Z |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Bundle/AddressingBundle/Form/EventListener/BuildAddressFormSubscriber.php [hunk 1]
```diff
@@ -137,6 +137,8 @@ private function createProvinceCodeChoiceForm(CountryInterface $country, $provin
         return $this->formFactory->createNamed('provinceCode', ProvinceCodeChoiceType::class, $provinceCode, [
             'country' => $country,
             'auto_initialize' => false,
+            'label' => 'sylius.form.address.province',
```
Comment (pamil) [inline src/Sylius/Bundle/AddressingBundle/Form/EventListener/BuildAddressFormSubscriber.php]: Are these really needed? These are the default values, both for `label` and `placeholder` (ProvinceCodeChoiceType is a child of ProvinceChoiceType defining those).

Comment (silverbackdan) [inline src/Sylius/Bundle/AddressingBundle/Form/EventListener/BuildAddressFormSubscriber.php]: When I define sylius.form.address.province to 'County' in a messages.yml, it will populate the ajax loaded province input, but the non-ajax loaded input still is labelled as 'Province Name'

Are you saying there's somewhere else I should be defining the labels instead of in the translation files?

The only reason I put this PR in is because these are the options on the inputs when loaded via ajax.

Perhaps it isn't needed for the province dropdown, my issue was only with the text input so it may be the problem is best fixed elsewhere.

Comment (pamil) [inline src/Sylius/Bundle/AddressingBundle/Form/EventListener/BuildAddressFormSubscriber.php]: @silverbackdan I guess the issue you encountered is fixed by adding label for `provinceName` field created in `createProvinceNameTextForm` (which is also done in this PR). Can you check if it is still fixed without this changes here?

Comment (silverbackdan) [inline src/Sylius/Bundle/AddressingBundle/Form/EventListener/BuildAddressFormSubscriber.php]: @pamil I had only extended it to the dropdown as well because I noticed they hadn't been defined in there either when they had been in the class handling realtime province field updates. The dropdown part is untested, only assumed. My issue is indeed fixed just by adjusting `createProvinceNameTextForm`

**Comment by pamil** [comment]
Thanks @silverbackdan! 🎉
